codecov.yml

```diff
@@ -1,6 +1,7 @@
 comment:
   layout: "diff"
   behavior: default
+  require_changes: true # Post coverage comment only if coverage changes
 
 coverage:
```

Comment:
In this case, the target is set to 95%, which means that the code coverage must be at least 95% for the project to pass the coverage check. I would recommend to keep it 80% or 85% ,threshold can be 10%.

Reply:
Thanks. I had misinterpreted how the settings were applied. I think 85% is reasonable as we are currently at 91% and I would like to get that higher.